src/Resources/config/doctrine/CreditMemo.orm.xml

```diff
@@ -18,6 +18,7 @@
         <field name="units" type="json" />
         <field name="currencyCode" />
         <field name="localeCode" />
+        <field name="comment" />
```

Comment:
Is this field required?

Reply:
For me `comment` is an optional field, so it could be nullable, but I'm not fully convinced by any of these approaches, so it could stay at it is right now. 
Wdyt @bartoszpietrzak1994 ?

Reply:
Making this field nullable may lead to state where you have both NULL and empty comments in the database. I would stay with the current solution :D

Reply:
Agreed 👍

Reply:
@bartoszpietrzak1994 Should be fixed now